cmd/profile/delete.go

```diff
@@ -24,7 +26,15 @@ func deleteProfileCommand() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to get k8s client config: %s", err)
 			}
-			return deleteProfile(config, ns, args[0])
+			profileName := args[0]
+			err = deleteProfile(config, ns, profileName)
+			if err != nil {
+				fmt.Printf("%s not found, assuming legacy profile\n", profileName)
+				// try deleting the legacy profile
+				errLegacy := deleteProfileLegacy(config, ns, profileName)
+				errors.CheckError(errLegacy)
```

Comment:
Does CheckError() terminate the program? For consistency, don't do this. Instead, just return the error.

Reply:
Yes, CheckError terminates the program, but I'll remove it.